build.gradle

```diff
@@ -56,30 +56,30 @@ shadowJar {
 }
 
 task prepareManifest(type: Copy) {
-    from 'launch4j/liteloader_installer_manifest.xml'
-    into tasks.launch4j.temporaryDir
+    from 'createExe/liteloader_installer_manifest.xml'
+    into tasks.createExe.temporaryDir
     expand(
         artefact:  project.sxsname,
         msversion: project.msversion
     )
 }
 
-launch4j {
+createExe {
     copyConfigurable = project.tasks.shadowJar.outputs.files
     jar              = 'lib/' + project.tasks.shadowJar.archiveName
-    icon             = "${projectDir}/launch4j/liteloader_installer_icon.ico"
-    manifest         = "${tasks.launch4j.temporaryDir}/liteloader_installer_manifest.xml"
+    icon             = "${projectDir}/createExe/liteloader_installer_icon.ico"
+    manifest         = "${tasks.createExe.temporaryDir}/liteloader_installer_manifest.xml"
     outfile          = tasks.shadowJar.archiveName - ".jar" + ".exe"
     version          = project.msversion
     internalName     = project.sxsname
     productName      = "LiteLoader"
-    description      = "LiteLoader Installer for Minecraft ${project.mcversion} release {project.verRelease}"
-    copyright        = "Copyright (C) 2013-15 Adam Mummery-Smith"
+    fileDescription  = "LiteLoader Installer for Minecraft ${project.mcversion} release {project.verRelease}"
+    copyright        = "Copyright (C) 2013-2018 Adam Mummery-Smith"
 }
 
 task gather(type: Copy) {
-    dependsOn 'launch4j'
-    from 'build/launch4j/' + launch4j.outfile
+    dependsOn 'createExe'
+    from 'build/createExe/' + createExe.outfile
```

Comment:
`createExe` still outputs into `build/launch4j`, not `build/createExe`. Also, probably cleaner to use gstrings than concatenation.

Reply:
Both Issues are now Fixes (Sorry it took so long!)